### PR TITLE
Feature/player class

### PR DESCRIPTION
**Pull request by Drakwrizz**
Added 1 player, Game Over screen, a simple map, a placeholder for the sprite.

### CODE REVIEW

File: GameOver.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using Godot;
+using System;
+
+public partial class GameOver : Node2D
+{
+	private void _on_button_pressed()
```
Comment (l0u1sg) [inline GameOver.cs:6]: Even if it's obvious, could you add the specification for this function?

File: GameOver.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	{
+		GetTree().ChangeSceneToFile("res://Map.tscn");
+	}
+	
+	private void _on_button_2_pressed()
```
Comment (l0u1sg) [inline GameOver.cs:11]: And also for this function?

**Review by l0u1sg — CHANGES_REQUESTED** [review]
Everything seems good for me, except that you have two functions that are uncommented. Even if their utility is obvious, it is better if it's commented

👍 for all the comments on your code ;)

File: GameOver.cs (hunk 2 above)
Comment (Drakwrizz) [reply GameOver.cs:11]: ok

File: GameOver.cs (hunk 1 above)
Comment (Drakwrizz) [reply GameOver.cs:6]: ok

**Review by l0u1sg — APPROVED** [review]

**Pull request merged**